### ISSUE TITLE
If VoiceOver is enabled, FlatList does not scroll when end of list is reached

### ISSUE THREAD

**Issue by aknad** [facebook/react-native#23140]
https://github.com/facebook/react-native/issues/13743

## Environment

```
System:
OS: macOS High Sierra 10.13.6
CPU: (4) x64 Intel(R) Core(TM) i5-4308U CPU @ 2.80GHz
Memory: 72.14 MB / 16.00 GB
Shell: 3.2.57 - /bin/bash
Binaries:
Node: 10.1.0 - ~/.nvm/versions/node/v10.1.0/bin/node
Yarn: yarn install v0.27.5
npm: 6.5.0 - ~/.nvm/versions/node/v10.1.0/bin/npm
Watchman: 4.1.0 - /usr/local/bin/watchman
SDKs:
iOS SDK:
Platforms: iOS 12.1, macOS 10.14, tvOS 12.1, watchOS 5.1
Android SDK:
API Levels: 22, 23, 24, 25, 26, 27
Build Tools: 23.0.1, 23.0.3, 25.0.0, 25.0.2, 25.0.3, 26.0.1, 26.0.2, 27.0.3, 28.0.3
System Images: android-23 | Android TV ARM EABI v7a, android-23 | Android TV Intel x86 Atom, android-23 | Android Wear ARM EABI v7a, android-23 | Android Wear Intel x86 Atom, android-23 | ARM EABI v7a, android-23 | Intel x86 Atom, android-23 | Intel x86 Atom_64, android-23 | Google APIs ARM EABI v7a, android-23 | Google APIs Intel x86 Atom, android-23 | Google APIs Intel x86 Atom_64, android-24 | Android TV Intel x86 Atom, android-24 | Intel x86 Atom, android-24 | Intel x86 Atom_64
IDEs:
Android Studio: 3.2 AI-181.5540.7.32.5056338
Xcode: 10.1/10B61 - /usr/bin/xcodebuild
npmPackages:
react: 16.6.3 => 16.6.3
react-native: 0.57.8 => 0.57.8
npmGlobalPackages:
create-react-native-app: 0.0.6
react-native-cli: 2.0.1
react-native-git-upgrade: 0.2.7
```

## Description

FlatList doesn't scroll when you reach the end of the list
I try to set _removeClippedSubviews={false}_ as suggested in the issue #13743 but it doesn't work

## Reproducible Demo
Here's an example:
https://snack.expo.io/@aknad/flatlist-accessibility

if you turn VoiceOver on and swipe listen the items, when you reach the end the FlatList doesn't scroll

**Comment by Salakar** [comment]
@aknad Thanks for reporting this 💛 

I've replicated the code you mentioned above on RN 0.59 and it was possible to scroll to the end with VoiceOver enabled:

![screenshot](https://cdn.discordapp.com/attachments/515573218376155156/557601535262195739/IMG_0037.PNG)

When VoiceOver is enabled hard pressing whilst swiping on the list enables you to scroll through the list.

I'll close this issue as I can no longer reproduce. Feel free to send a reproduction repo on a new issue if this is still a problem for you.

Thank you

**Comment by saharbit** [comment]
Still not working for me on android 9, Expo 31 and RN 0.57.
I have tried multiple setups as well as your snack code, but the list still doesn't scroll as i swipe, it just stops when the screen is no longer visible and restart to the first item.

`<FlatList
          data={[
            {title: 'Title 1'},
            {title: 'Title 2'},
            {title: 'Title 3'},
            {title: 'Title 4'},
            {title: 'Title 5'},
            {title: 'Title 6'},
            {title: 'Title 7'},
            {title: 'Title 8'},
            {title: 'Title 9'},
            {title: 'Title 10'},
            {title: 'Title 11'},
            {title: 'Title 12'},
            {title: 'Title 13'},
            {title: 'Title 14'},
            {title: 'Title 15'},
            {title: 'Title 16'},
            {title: 'Title 17'},
            {title: 'Title 18'},
            {title: 'Title 19'},
            {title: 'Title 20'},
          ]}
          renderItem={({item, index}) => {
            return(
            <View 
              style={styles.item}
              accessible="true"
              ref={(ref) => {
                this.itemsRefs[index] = ref;
              }}
            >
              <Text>{item.title}</Text>
            </View>)
          }}
          keyExtractor={item => item.id}
          removeClippedSubviews={false}
        />`
